algorithms/aqc/solving_qlsp/solving_qlsp_with_aqc.ipynb

```diff
@@ -0,0 +1,867 @@
+{
```

Comment:
<p>I would also add a zoom-in picture to the y values [-1.3, 1.3]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/688/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>